src/ocean/io/compress/Lzo.d

```diff
@@ -243,6 +243,23 @@ class Lzo
     }
 }
 
+/// Simple compress / decompress example.
+unittest
+{
+    auto lzo = new Lzo;
+    auto original = "compress this!".dup;
+
+    // Compress.
+    auto compressed = new void[lzo.maxCompressedLength(original.length)];
+    compressed.length = lzo.compress(original, compressed);
+
+    // Decompress (with overrun safety checks).
+    auto uncompressed = new void[original.length];
+    lzo.decompressSafe(compressed, uncompressed);
```

Comment:
Test that it returns `uncompressed.length`.

Reply:
As this is primarily intended to be a usage example (documented unittest), I'm not sure about that. In the example given, the return value is irrelevant, as we've already ensured that the decompress destination buffer is exactly the right size.

Reply:
Maybe add a comment, otherwise it may look as if the return value is irrelevant.